doc/source/conf.py

```diff
@@ -157,17 +157,25 @@
         "image_light": "logo.png",
         "image_dark": "logo_dark.png",
     },
+    # https://pydata-sphinx-theme.readthedocs.io/en/stable/user_guide/header-links.html#fontawesome-icons
+    "github_url": "https://github.com/arviz-devs/arviz",
+    "twitter_url": "https://twitter.com/arviz_devs",
     "icon_links": [
+        # {
+        #     "name": "GitHub",
+        #     "url": "https://github.com/arviz-devs/arviz",
+        #     "icon": "fab fa-square-github",
+        # },
+        # {
+        #     "name": "Twitter",
+        #     "url": "https://twitter.com/arviz_devs",
+        #     "icon": "fab fa-square-twitter",
+        # },
         {
-            "name": "GitHub",
-            "url": "https://github.com/arviz-devs/arviz",
-            "icon": "fab fa-github-square",
-        },
-        {
-            "name": "Twitter",
-            "url": "https://twitter.com/arviz_devs",
-            "icon": "fab fa-twitter-square",
-        },
+            "name": "Mastodon",
+            "url": "https://fosstodon.org/@ArviZ@bayes.club",
+            "icon": "fab fa-mastodon",
+        }
     ],
```

Comment:
How about:

```
"icon_links": [
        {
            "name": "GitHub",
            "url": "https://github.com/arviz-devs/arviz",
            "icon": "fa-brands fa-square",
        },
        {
            "name": "Twitter",
            "url": "https://twitter.com/arviz_devs",
            "icon": "fa-brands fa-twitter",
        },
            "name": "Mastodon",
            "url": "https://bayes.club/@ArviZ",
            "icon": "fa-brands fa-mastodon",
        }
```

which generates:

![imatge](https://user-images.githubusercontent.com/23738400/203581002-1e348b03-0560-46a6-b29f-92ca0e638d8d.png)


If we want to have twitter be blue we can add the css manually. We originally had the shortcuts, but their usage and encouragement on using them from the theme has been irregular, at some point it looked like the shortcuts might even be removed. I feel more comfortable with the explicit icons for now (you might have already noticed the theme is nice but still very new and unstable both look-wise and use-wise)

Side note, I also updated the mastodon link to be directly to the right space.

Reply:
Ooh thanks! This is way better and 100% agree with your comments. I like keeping everything the same color so that one doesn't stand out over the others.